documentation/src/main/asciidoc/topics/protobuf/library.proto

```diff
@@ -1,3 +1,4 @@
+//needs review
```

Comment:
@fax4ever Can you please take a look and update this example?

Reply:
Yes we need to upgrade with the new annotations. I missed it last time :)

Reply:
With the new annotations it becomes:

``` proto
package book_sample;

/* @Indexed */
message Book {

    /* @Text(projectable = true) */
    optional string title = 1;

    /* @Text(projectable = true) */
    optional string description = 2;

    // no native Date type available in Protobuf
    optional int32 publicationYear = 3; 

    repeated Author authors = 4;
}

message Author {
    optional string name = 1;
    optional string surname = 2;
}
```